cmd/minishift/cmd/start.go

```diff
@@ -56,35 +57,6 @@ import (
 
 const (
 	commandName = "start"
-
-	// minishift
-	isoURL                = "iso-url"
```

Comment:
nice, I like that we get rid of this here. Less lines in start.go. Together with #962 start.go start.go looks manageable again.